docs/src/main/sphinx/functions.rst

```diff
@@ -2,8 +2,8 @@
 Functions and operators
 ***********************
 
-This chapter describes the built-in SQL functions and operators supported by
-Trino. They allow you to implement complex functionality and behavior of the SQL
+This section describes the built-in SQL functions and operators supported by
+Trino. They allow you to implement complex capabilities and behavior of the SQL
```

Comment:
The wording here is better, but still a little awkward. "implement behavior of the SQL executed by Trino" doesn't read all that well. Not a huge deal because it's already this way, but maybe fix it up while we're here?

Reply:
Any suggestion?

Reply:
"They allow you to implement complex capabilities and behaviors to the SQL executed by Trino..." would be a big step up.

Reply:
changed SQL to queries .. sounds better now imho